src/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
The selector `&--4 :nth-child(-n + 4)` might be too specific and could lead to unexpected styling if not used carefully. Ensure that this class is applied correctly in the HTML to avoid styling issues. Consider using a more descriptive class name if possible.